wordclock/http_server.c

```diff
@@ -582,14 +602,14 @@ static void handle_wifi_station_post(int s, wificfg_method method,
                     if (ssid[0] != '\0' && password[0] != '\0') {
                         wificfg_write_string(s, "Rebooting with new WIFI settings\r\n");
                         closesocket(s);
-                        Sleep(1000);
                         struct sdk_station_config config = {"", "", 0, {0}};
                         strncpy((char*)config.ssid, ssid, sizeof(config.ssid));
                         strncpy((char*)config.password, password, sizeof(config.password));
-                        sdk_wifi_set_opmode(STATION_MODE);
+                        // sdk_wifi_set_opmode(STATION_MODE);
```

Comment:
Weet je heel zeker dat dit in alle corner cases goed gaat?
Rond de stage change van en naar AP mode heb ik laatst heel veel getest.

Reply:
Waarom zou dat niet goed gaan? Er wordt een restart gedaan. Ik had juist het idee dat het niet goed ging zoals het zat. De mode veranderen en dan meteen een restart doen. Dat voelde ook niet goed.

Reply:
nu zie ik hier nog wat fuzzy gedrag. Ik test nog ff wat

Reply:
Ik heb hem voor de zekerheid maar teruggezet